apiserver/README.md

```diff
@@ -0,0 +1,33 @@
+
+
+kubernetes genericapiserver base
+
+Start with:
```

Comment:
We need to vendor the dependencies required for this.  Do not rely on system GOPATH.